### PR TITLE
[MIG][10.0] auto_backup

### PR DESCRIPTION
**Pull request by lasley**
* Change openerp to odoo
* Bump version
* Rename manifest
* Add pysftp to requirements

### CODE REVIEW

File: auto_backup/README.rst
Comment (lasley) [inline auto_backup/README.rst:27]: We ran into an issue where backup execution was dying when running inside of Odoo due to the timeouts configured on the instance. We had to make a small script to run this system outside of the main Odoo instance to circumvent this, which is described [in this blog post](https://blog.laslabs.com/2016/10/running-python-scripts-within-odoos-environment/).

If this is something other people have run into as well, maybe transferring some of that info to this ReadMe would be of use?

**Comment by max3903** [comment]
@lasley Why don't you add a link to your blog post in the module description (know issues section) ?

**Comment by lasley** [comment]
Good call, that's exactly where it should go!

**Comment by yajo** [comment]
Why did we lose commits?

**Comment by lasley** [comment]
Squash and merge? The cherry-picked is definitely the commit for this on 9.0

**Comment by yajo** [comment]
I think the common strategy is to preserve others' commits (except those from transbot, that get squashed) when migrating versions. See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0

It has the advantage that all previous contributors get subscribed to the PR automatically. They're probably interested.

**Comment by lmignon** [comment]
@lasley Where is the commit histrory. We must preserve the history! The history is now empty also in 9.0.

**Review by lmignon — CHANGES_REQUESTED** [review]
The commit history from 8.0 must be preserved

**Comment by lasley** [comment]
Alright I did the required git-fu for fixing the commit history after merge of #620. Better?

**Review by lmignon — APPROVED** [review]
code review only

**Review by max3903 — APPROVED** [review]

**Pull request merged**
